to-do-list.css

```diff
@@ -69,6 +69,6 @@ input[type="text"], textarea {
   text-decoration: line-through;
 }
 
-.completed, .uncompleted {
+.task-item, .uncompleted {
```

Comment:
you do not need `uncompleted` class

Reply:
This is what we have discussed on messenger

Reply:
Forgot about this one. Deleted it already.